app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,11 @@
+class IngredientsController < ApplicationController
```

Comment:
You don't need this controller for the user stories on this IC, but if I wanted to create a new ingredient, this would be a great place to put the functionality!